### PR TITLE
feat: added home page for e-commece website

### PR DESCRIPTION
**Pull request by SanjayJatti**
- Header
- Hero section with background image
- New arrival section containing horizontal cards
- Footer
home-e-commerce

### CODE REVIEW

**Comment by vaibhav18matere** [comment]
you can reduce image opacity, the text on it will be visible clearly

File: home.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    border-radius: 0.3rem;
+}
+.text-sec{
+    color: var(--secondary-color);
+    font-size: inherit;
+}
+.new-arrival-section{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
```
Comment (vaibhav18matere) [inline home.css]: you can make separate class for this

**Review by vaibhav18matere — APPROVED** [review]

File: home.css (hunk 1 above)
Comment (SanjayJatti) [reply home.css]: 960b352b82f8feb1b700e7553d0136591bf33f14

**Review by prachi-sahani — COMMENTED** [review]
code looks fine 👍🏻

**Review by prathmeshjagtap — APPROVED** [review]

**Pull request merged**